src/python/nimbusml/tests/ensemble/test_ensembleregressor.py

```diff
@@ -33,7 +33,7 @@ def test_ensembleregressor(self):
         scores = ensemble.predict(X_test)
 
         r2 = r2_score(y_test, scores)
-        assert_greater(r2, 0.12, "should be greater than %s" % 0.12)
+        assert_greater(r2, 0.105, "should be greater than %s" % 0.105)
```

Comment:
>.105,  [](start = 28, length = 6)

Numerical changes are due to default see change. It appeared that seed was passed in 2 params (seed & random_state) to ML.NET , one was always and another always 42. This change removes seed and uses only random_state with default 42.